apache/zeppelin
# [ZEPPELIN-2257] notification about incompleteness of data

### What is this PR for?
Added notification about exceeding the limit in the result.

### What type of PR is it?
Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-2257

### How should this be tested?
1. Create table *test* with more than 2 records (if not exists)
2. Set parameter `common.max_count ` = 2
3. Execute 
```
%jdbc
select *from test
```
You should see message about exceeds limit

### Screenshots (if appropriate)
![6](https://cloud.githubusercontent.com/assets/25951039/23899435/f5a23e48-08d6-11e7-9cb0-1613398ce22e.png)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no
